packages/react-composites/src/composites/CallComposite/pages/CallPage.tsx

```diff
@@ -9,7 +9,7 @@ import { ActiveErrorMessage, ErrorBar, ParticipantMenuItemsCallback } from '@int
 import { VideoGalleryLayout } from '@internal/react-components';
 import React from 'react';
 /* @conditional-compile-remove(dtmf-dialer) */
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
```

Comment:
this useEffect may cause an error if it's only being used in a conditionally compiled way in the rest of the file.